{"name": "test_simple", "status": "failed", "statusDetails": {"message": "AssertionError: assert 4 == 5", "trace": "self = <test_login_page.TestLoginPage object at 0x7f391a8ae1b0>\n\n    def test_simple(self):\n        self.login_page.open()\n        self.login_page.enter_login(\"123\")\n>       assert 4 == 5\nE       assert 4 == 5\n\ntests/test_login_page.py:24: AssertionError"}, "start": 1746729298665, "stop": 1746729299533, "uuid": "bebaf3b2-d79d-4d73-b300-344bb78decf4", "historyId": "b625b33df7b710c19b0b63ad85e53788", "testCaseId": "b625b33df7b710c19b0b63ad85e53788", "fullName": "tests.test_login_page.TestLoginPage#test_simple", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_page"}, {"name": "subSuite", "value": "TestLoginPage"}, {"name": "host", "value": "17319d71a2f1"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_page"}]}